test/test_torch.py

```diff
@@ -1653,14 +1655,6 @@ def reference(x, k, o3, o32):
         self._test_conv_corr_eq(lambda x, k: torch.xcorr3(x, k), reference)
 
     @unittest.skip("Not implemented yet")
-    def test_xcorr3_xcorr2_eq(self):
```

Comment:
This test is not the same as [this one, which has the same name](https://github.com/lukeyeager/pytorch/blob/591a25490732088c9bb5bc11835af041bc58a9e9/test/test_torch.py#L1650-L1655). I think we should change the name instead of removing it.